ProjFS.Mac/Scripts/Build.sh

```diff
@@ -67,6 +67,7 @@ while read line; do
 	 [[ $line != *"FindOrDetectRootAtVnode"* ]] &&                   #SHOULD ADD COVERAGE
 	 [[ $line != *"FindUnusedIndexOrGrow_Locked"* ]] &&              #SHOULD ADD COVERAGE
 	 [[ $line != *"FindRootAtVnode_Locked"* ]] &&                    #SHOULD ADD COVERAGE
+	 [[ $line != *"DirectoryContainsPath"* ]] &&                     #TODO: Why does this get reported as not having coverage when it does?
```

Comment:
You probably know this already but after generating a code coverage run, xcode has a really nice display for seeing sections it believes are not covered.

Reply:
The coverage counter column in the code view? Yes, this was showing nonzero on all branches, and I just wanted to run the functional tests rather than leaving it as I didn’t have time to investigate the discrepancy last night.